lib/rules/function-whitespace-after/index.js

```diff
@@ -75,7 +75,7 @@ const rule = (primary, _secondaryOptions, context) => {
 					return;
 				}
 
-				if (source.substr(index, 2) === '\r\n') {
+				if (source.slice(index, index + 2) === '\r\n') {
```

Comment:
[note] Since the `index` variable is not necessarily `0`, it's needed to change the second argument.